openmdao/docs/index.rst

```diff
@@ -10,6 +10,7 @@ OpenMDAO Documentation
     user_guide/index.rst
     feature_reference/index.rst
     examples/index.rst
+    api_translation/index.rst
```

Comment:
Thanks for the fixes...the one thing missing...I see the old version of the doc being "removed" but don't see the index.rst being "added"

Reply:
Fixed